addons/viewrestriction/XEH_clientInit.sqf

```diff
@@ -0,0 +1,34 @@
+#include "script_component.hpp"
+
+// Exit on Headless Client
+if !(hasInterface) exitWith {};
+
+// Exit if third person view is not available
+if !(difficultyEnabled "3rdPersonView") exitWith {};
```

Comment:
Add some RPT log if it's any of the mods are enabled but not available due to this line.